packages/lib-react-components/src/PlainButton/PlainButton.js

```diff
@@ -18,24 +18,26 @@ export const StyledPlainButton = styled(Button)`
 
 function PlainButton (props) {
   const {
-    as,
-    className,
-    disabled,
-    href,
-    onClick,
-    labelSize,
-    text,
-    color,
+    as = '',
+    className = '',
+    disabled = false,
+    href = '',
+    onClick = () => {},
+    labelSize = 'medium',
+    text = '',
+    color = {
+      dark: 'accent-1',
+      light: 'neutral-1'
+    },
```

Comment:
This generates a new object on each render. It's better to define the object once:
```js
const defaultColor = {
  dark: 'accent-1',
  light: 'neutral-1'
}
```
then use that as the default value for the destructured argument:
```js
color = defaultColor
```

Reply:
Just one of JavaScript's little quirks. Would it be worth enabling strict mode, so that we get automated warnings from React for things like this?

Reply:
Sorry, one other change: the destructuring here should happen inside the function declaration. Right now, the storybook isn't picking up any of these default values.

```js
function PlainButton({
  as = '',
  className = '',
  disabled = false,
etc.
})
```